woeip/apps/air_quality/models.py

```diff
@@ -54,19 +54,20 @@ def __str__(self):
 
 
 class SessionData(TimeStampedModel):
-    """The raw data file generated during a session. Assumes one and only one file per sensor,
-    although multiple sensors can be linked to one session"""
-    upload = models.FileField(upload_to='session_data')
+    """The raw data files generated during a session. Assumes one general and one gps file.
+    Multiple sensors can be linked to one session"""
+    sensor_file = models.FileField(upload_to='sensor_files', default="")
```

Comment:
At the risk of slowing things down, it's probably worth having one more chance to hash out the schema before we commit. I admit I didn't communicate very clearly when designing it, so any misunderstanding is my bad. But I think we can get something will a) work for most future data sources, and b) not be too cumbersome for the dustrak/gps data source that is the only one we currently have.

Reply:
Here's what I'm thinking:
https://www.lucidchart.com/invitations/accept/5adb4594-1f61-4230-936f-32b522302382

I drew two cases. The top is the one we actually have, one device that records air quality and one that records GPS. Each produces their own raw data file. The bottom one is another (I think likely) case, one device that records air quality and GPS location and produces a single raw data file.

Reply:
Also realize this is a bit of a detour from the current PR. Down to merge this since it contains a lot of awesome front end and form handling parts, and address the DB schema in a separate issue. In that case, we would still have to rewrite some of the DB parts of this PR, but could be worth it to keep moving forward.

Reply:
Agree with @r-b-g-b that this schema of the table is something we'll likely have to revisit later, and also that it makes sense to move forward with this PR as it stands now. 
It seems there's still a little ambiguity on what will make the most sense for the schema, and I think it will become much clearer as we start testing the data flow. So even if we revise the schema, committing to this solution now is a step forward.

Reply:
Sweet, in that case I'll open a new issue and approve this request. I'll say don't worry about the migration comment I made earlier.